.github/workflows/icu_ci.yml

```diff
@@ -32,6 +32,27 @@ jobs:
           ant init;
           ant check;
 
+  # Out of source build with gcc 8 and Wextra.
+  gcc-8-Wextra:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: ICU4C with gcc 8 and extra warnings.
+        run: |
+          mkdir build;
+          cd build;
+          ../icu4c/source/runConfigureICU Linux --disable-debug --enable-release --disable-layoutex --prefix=$PREFIX;
+          make -j2;
+          make -j2 check;
+          make install;
+          PATH=$PREFIX/bin:$PATH make -C test/hdrtst check
+        env:
+          PREFIX: /tmp/icu-prefix
```

Comment:
Optional: I assume that the order of "run" vs. "env" doesn't matter, but for human understanding I would prefer "env" before "run".

Reply:
Rearranged.